packages/console/src/pages/ApplicationDetails/components/InputWithDelete/index.module.scss

```diff
@@ -0,0 +1,18 @@
+@use '@/scss/underscore' as _;
+
+.inputWithDelete {
+  display: flex;
+
+  .textField {
+    @include _.form-text-field;
+  }
+
+  .removeButton {
+    margin-left: _.unit(2);
+    cursor: pointer;
+
+    svg {
+      height: 100%;
+    }
```

Comment:
better keep the SVG it's original size or give it a numeric size

Reply:
Rebased from master and we use the `IconButton` component.